src/main/cli/cli.c

```diff
@@ -4845,12 +4845,13 @@ static void cliRcSmoothing(const char *cmdName, char *cmdline)
     if (rxConfig()->rc_smoothing_mode) {
         cliPrintLine("FILTER");
         if (rcSmoothingAutoCalculate()) {
-            const uint16_t avgRxFrameUs = rcSmoothingData->averageFrameTimeUs;
-            cliPrint("# Detected RX frame rate: ");
-            if (avgRxFrameUs == 0) {
+            const uint16_t smoothedRxRateHz = lrintf(rcSmoothingData->smoothedRxRateHz);
+            int rxIntervalUs = getCurrentRxIntervalUs();
+            cliPrint("# Detected Rx frequency: ");
+            if (rxIntervalUs == 0) {
                 cliPrintLine("NO SIGNAL");
             } else {
-                cliPrintLinef("%d.%03dms", avgRxFrameUs / 1000, avgRxFrameUs % 1000);
+                cliPrintLinef("%dHz", smoothedRxRateHz);
```

Comment:
```suggestion
                cliPrintLinef("%.0fHz", rcSmoothingData->smoothedRxRateHz);
```

Reply:
Very cool, much better!  Didn't know about that method.

Reply:
Hmm @KarateBrot I get this error:
```
./src/main/cli/cli.c:4852:56: error: implicit conversion from 'float' to 'doubl' when passing argument to function [-Werror=double-promotion]
 4852 |                 cliPrintLinef("%.0fHz", rcSmoothingData->smoothedRxRateHz);
 ```